hbase_sql/src/main/resources/core-site.xml

```diff
@@ -36,7 +36,12 @@
 
     <property>
         <name>fs.defaultFS</name>
-        <value>file:///</value>
```

Comment:
I would split the filesystem change into a different issue, it has nothing to do with LocatedRow.